packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -474,11 +474,14 @@ const LoadTestDataStoreInstantiationFactory = new DataObjectFactory(
     {},
 );
 
+const innerRequestHandler = async (request: IRequest, runtime: IContainerRuntimeBase) =>
+    runtime.IFluidHandleContext.resolveHandle(request);
```

Comment:
fyi @vladsud 

i don't think we want to encourage code like this

Reply:
it should pass even by injecting `rootDataStoreRequestHandler` instead of `innerRequestHandler`

Reply:
I don't think a root request works here, as the data store it's requesting isn't a root (in `getGCDataStore`).

Reply:
Chatted with Tony offline about this -- we want to minimize the calls to `resolveHandle`, but `innerRequestHandler` was already doing making these calls.

The strategy we are taking to eliminate `resolveHandle` calls is to first remove the "baked in" calls in our base runtime factories to minimize by-default availability (prevent folks accidentally taking dependencies on the bad practice), and then longer-term we should move toward deprecating/removing the public exposure of `resolveHandle`.